ApplicationPattern.yaml

```diff
@@ -9439,7 +9333,15 @@ components:
                 type: string
                 description: >
                   'Request that shall be called, when button gets pressed
-                  from [/core-model-1-4:control-construct/profile-collection/profile=*-1-0-0-action-p-*/action-profile-1-0:action-profile-pac/action-profile-configuration/request]'
+                  from
+                  [{/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-*/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-protocol}]
```

Comment:
Kindle please clarify my doubt. Whether we have consider 1-0-0-tcp-s-bm-000 as a default value here ? or either from the request-body/header we have to figureout what protocol needs to be sent in the response.

Reply:
Because we (willingly) did not make any decision about how and where to communication the required-protocol or zone, I am proposing to consider 1-0-0-tcp-s-bm-000 as a default value.

Reply:
Thanks for the clarification.